packages/dcos-sdk/README.md

```diff
@@ -0,0 +1,52 @@
+# DC/OS UI SDK
+
+DC/OS plugin developer's best friend!
+
+## NavigationService
+
+ExtensionProvider that provides extension point for navigation.
+
+Most probably you don't want to use it directly but rather implement its `INavigationExtension` and bind it to `ExtensionProvider` named `NavigationExtensionProvider`.
```

Comment:
What do you think of enhancing the example to show how to do the bindings?

Reply:
I could 👍 though using the library assumes one familiar with inversify.js decorators.

Reply:
That

Reply:
That's also fine, then please state that somewhere and add a link for more information :)